crates/re_space_view_spatial/src/eye.rs

```diff
@@ -278,35 +391,41 @@ impl OrbitEye {
             } else if response.dragged_by(ROTATE3D_BUTTON) {
                 self.rotate(response.drag_delta());
             } else if response.dragged_by(DRAG_PAN3D_BUTTON) {
-                self.translate(response.drag_delta());
+                let delta_in_view = 0.001 * speed * response.drag_delta(); // TODO(emilk): take fov and screen size into account?
```

Comment:
would find fovY/fovZ/screensize to be strange modifiers for this, but maybe that's just taste

Reply:
`0.001` is approximately `1/screensize` so that panning one ui-point in orbit mode feels like moving the camera by that amount

Reply:
I added a comment explaining it

Reply:
💡